src/main/java/frc/robot/RobotContainer.java

```diff
@@ -29,6 +29,25 @@
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import edu.wpi.first.wpilibj2.command.button.POVButton;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
+import frc.robot.commands.CameraDriveCommand;
+import frc.robot.NetworkTables.NetworkTablesBase;
+import frc.robot.commands.AutoShoot;
+import frc.robot.commands.CMD_ShooterManualRPM;
+import frc.robot.commands.CMD_canalThrough;
+import frc.robot.commands.CMD_changeSetpoint;
+import frc.robot.commands.CanalZeroToOneBottom;
+import frc.robot.subsystems.IndexSubsystem;
+import frc.robot.subsystems.Climber;
```

Comment:
in robot container this wall of imports is cleaned up using:
import frc.robot.commands.* and import frc.robot.subsystems.*

Reply:
fixed it